vaccine_feed_ingest/runners/us/carbon_health/fetch.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import requests
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+output_dir = sys.argv[1]
+if output_dir is None:
+    logger.error("Must pass an output_dir as first argument")
+    sys.exit(1)
+
+base_url = "https://carbonhealth.com/static/data"
+
+latest = requests.get(f"{base_url}/rev-manifest.json").json()["covid-vaccine.json"]
```

Comment:
Look into `urljoin` for joining path parts. It will ensure the URL is joined correctly: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin

Reply:
Also, can you split about the result parsing and check for a successful response before grabbing the results. It will be helpul if this breaks to get an error log to help with debugging what went wrong.